DashWallet/Sources/UI/Payments/Amount/SendAmountViewController.swift

```diff
@@ -57,7 +57,7 @@ class SendAmountViewController: BaseAmountViewController {
         let account = DWEnvironment.sharedInstance().currentAccount
         let allAvailableFunds = account.maxOutputAmount
         
-        if model.amount.plainAmount > allAvailableFunds - CrowdNode.minimumLeftoverBalance {
+        if model.amount.plainAmount + CrowdNode.minimumLeftoverBalance > allAvailableFunds {
```

Comment:
Another case of arithmetic overflow is when a really small amount is sent. Unrelated to the one discussed.